src/core/src/CoreMain.py

```diff
@@ -115,6 +122,13 @@ def __init__(self, argv):
             composite_logger.log_debug("Completed exception handling.\n")
 
         finally:
+            # clean up temp folder after all operation execution is finished from Core
+            # Note to reviewer: This only deletes all *.list files currently
```

Comment:
Nit: rewrite full comment (to remove 'note to reviewer' also) -- "# clean up temp folder of files created by Core after execution completes"

Reply:
Resolved